src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300px;
+  font-family: inherit;
+  color: #3d4e61 ;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  outline: none;
+}
+
+.input--big {
+  height: 70px;
+  background-position: 26px 25px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
```

Comment:
set it for `.input`

![image](https://user-images.githubusercontent.com/55200530/224302260-c2da601d-e9cd-4278-a275-92bd691617c4.png)